### DISCUSSION TITLE
Change value of Prev and Next boxes

### DISCUSSION THREAD

**Discussion by teresalves** [thuliteio/doks#1267] [Q&A]
Hey everyone!
So there are the Prev and Next buttons at the end of each page that tell you the name of the next page. However, it is inside a certain Section. In the case of the print I send in attachment, the Next button points me to something called
 **"Basic Commands"** 
but I would like it to be 
**"Kubernetes - Basic Commands"** or something similar

<img width="1100" alt="Screenshot 2024-08-08 at 21 54 46" src="https://github.com/user-attachments/assets/720afba9-82e8-4f6b-9e3c-eb5b216799bc">

Could I maybe just add a parameter called "supertitle:" or something in the .md for that "Basic Commands" file so that this Next has a different value? How would I change the template then?

Thanks in advance!

**Comment by h-enk** [comment]
Hello Teresa! You could "automate" this in a template.

Copy `node_modules/@hyas/doks-core/layouts/partials/main/docs-navigation.html` and paste it as `layouts/partials/main/docs-navigation.html`.  Make your changes in the latter.

For example make lines 18 and 31 something like:

```html
<a href="{{ .RelPermalink }}" class="stretched-link text-reset text-decoration-none">{{ path.Base .File.Dir | humanize }} - {{ .Title }}</a>
```
Result:

![2024-08-09_09-02-36](https://github.com/user-attachments/assets/c24e89bd-fbc3-4477-bb65-1004eea8068a)

**Comment by teresalves** [comment]
Hey @h-enk ! Thanks so much for the quick reply, that works great!

But what if I actually had the "supertitle" next to the "title" in the md? How could I use that instead? 
I tried to see if it matched automatically by doing .Supertitle in that new layout file but that did not work.

Thanks again, I really appreciate it ❤️

**Reply by h-enk** [reply]
Display `supertitle` if it exists, else fallback to `title`:

```html
<a href="{{ .RelPermalink }}" class="stretched-link text-reset text-decoration-none">{{ with .Params.supertitle }}{{ . }}{{ else }}{{ .Title }}{{ end }}</a>
```

![2024-08-12_14-30-40](https://github.com/user-attachments/assets/f24f2c99-2cf0-4b4f-ba62-49246cedc36d)

**Reply by teresalves** [reply]
It works! Amazing, thanks again ❤️
